packages/shared/src/utils/constants/mt5-login-list-status.ts

```diff
@@ -6,4 +9,16 @@ export const MT5_ACCOUNT_STATUS = Object.freeze({
     PENDING: 'pending',
     POA_PENDING: 'poa_pending',
     POA_VERIFIED: 'poa_verified',
+    UNDER_MAINTENANCE: 'under_maintenance',
+    UNAVAILABLE: 'unavailable',
+});
+
+/**
+ * this comes from trading_platform_status endpoint
+ */
+export const TRADING_PLATFORM_STATUS = Object.freeze({
```

Comment:
Having `Object.freeze` it's not necessary any more. You can just append `as const` at the end of the object it will do the trick.